decimal.md

```diff
@@ -0,0 +1,53 @@
+---
+title: DECIMAL
+toc: true
+---
+
+## Description
+
+The `DECIMAL` [data type](data-types.html) stores exact, fixed-point numbers with no limit on digits to the left and right of the decimal point. This type is used when it is important to preserve exact precision, for example, with monetary data. 
+
+## Synonyms
+
+In CockroachDB, the following are synonyms of `DECIMAL` and are implemented identically:
```

Comment:
The `and are implemented identically` phrase used here and elsewhere seems to be implied by `synonym`. Do you think the additional detail is useful?